voltaire_bundler/cli_manager.py

```
@@ -1145,15 +1155,23 @@ async def get_init_data(args: Namespace) -> InitData:
     # operator's first request.
     try:
         await latest_block_cache.warm(ethereum_node_urls_rearranged)
+        logging.info("Latest-block cache warmed.")
     except Exception as exc:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)